src/pages/Improbability/Improbability.component.tsx

```diff
@@ -0,0 +1,69 @@
+import Typography from '@mui/material/Typography';
+import { useForm } from 'react-hook-form';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import { DataTable } from '../../components/DataTable/DataTable.component';
+import { PageView } from '../../components/PageView/PageView.component';
+import {
+  ImprobabilityComponentProps,
+  ImprobabilityFormValues,
+  ImprobabilityRowData,
+} from './Improbability.types';
+import { ControlledDropdown } from '../../components/Dropdown/Dropdown.controlled';
+import {
+  coinsListColumns,
+  improbabilityDropdownOptions,
+} from './Improbability.utils';
+
+export const ImprobabilityComponent = ({
+  state,
+  coins,
+}: ImprobabilityComponentProps) => {
+  const { setValue, control } = useForm<ImprobabilityFormValues>({
+    mode: 'onChange',
+    defaultValues: { network: '' },
+  });
+
+  return (
+    <PageView
+      title={
+        <Typography variant="h2" padding={1}>
+          Improbability Drive
+        </Typography>
+      }
+    >
+      <Box
+        sx={{
+          width: '100%',
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+          gap: 2,
+        }}
+      >
+        <ControlledDropdown
+          name="network"
+          placeholder="Choose Network"
+          control={control}
+          options={improbabilityDropdownOptions}
```

Comment:
This cane be done in the next MR, as separate feature.

Please update the options list so that:
- when user is connected to some testnet chain - only show testnet chains in the dropdown (e.g. `rskTestnet`, `Ropsten`)

- when connected to one of the production chains - show production chains in the dropdown (e.g. `RSK`, `BNB Chain`)